libs/agno/agno/team/team.py

```diff
@@ -1206,9 +1237,10 @@ async def _arun(
         7. Log the team run
         """
         self.model = cast(Model, self.model)
+
         # Resolving here for async requirement
-        if self.dependencies is not None:
-            await self._aresolve_run_dependencies()
+        if dependencies is not None:
```

Comment:
Your not resolving `dependencies` with `self.dependencies` like on `run`
Your not resolving the `add_history_to_context` or `add_dependencies_to_context` like you do for agent.